.gitignore

```
@@ -7,7 +10,7 @@ build/
 DerivedData/
 
 ## Documentation
-docs/
+docs/public_documentation
```

Just a suggestion, not critical.
```suggestion
docs/output
```

---

Updated output dir's name to `jazzy_output` as you recommend.